### PR TITLE
MSA-10803 : Cisco FirePower Adapter

### PR DESCRIPTION
**Pull request by UBiqube-ydu**

### CODE REVIEW

**Comment by UBiqube-ydu** [comment]
Work done:
- MS stuff
- backup/restore
- provisioning

Work to do:
- asset
- firmware update
- license update

File: adapters/cisco_ftd_rest/conf/device.properties [hunk 1]
```diff
@@ -0,0 +1,7 @@
+model.id = 2070001
+model.name = FTD
+manufacturer.id = 1
+manufacturer.name = Cisco
+manufacturer.category = Security
```
Comment (abr-ubiqube) [inline adapters/cisco_ftd_rest/conf/device.properties:5]: all the other Cisco are in category Network

Comment (UBiqube-ydu) [reply adapters/cisco_ftd_rest/conf/device.properties:5]: Oh I saw that and I fixed on my MSA but forgot in source !

Comment (abr-ubiqube) [reply adapters/cisco_ftd_rest/conf/device.properties:5]: but we can put FTD in Security, I think @ubihgu may have an opinion on this

**Review by abr-ubiqube — APPROVED** [review]
a full review would probably require some real testing

**Pull request merged**